cypress/support/ethereum.ts

```diff
@@ -3,19 +3,17 @@
  */
 
 import { Eip1193Bridge } from '@ethersproject/experimental/lib/eip1193-bridge'
-// eslint-disable-next-line @typescript-eslint/no-restricted-imports
```

Comment:
Why is this being removed? Is this no longer being linted against?

Reply:
Yeah linter no longer throws an error/warning

Reply:
perfect use case for `@ts-expect-error`